NEWS.md

```diff
@@ -1,3 +1,8 @@
+## [4.37]
+- New functions for new role and permissions system (#114)
```

Comment:
Maybe also include details on any breaking changes or argument ordering changes, if there is some. In any event, we should warn that legacy roles will no longer be supported from release 4.37+